Dockerfile

```diff
@@ -1,7 +1,16 @@
 #base stage
 FROM archlinux:base-devel AS builder
 
-RUN pacman -Syu rustup mold musl rust-musl --noconfirm
+# Force-sync packages, install archlinux-keyring, repopulate keys
+RUN pacman -Syy
+RUN pacman -S archlinux-keyring --noconfirm --disable-download-timeout
```

Comment:
I would have thought that this isn't necessary on `base-devel`. How come *nobody* has good docker support.

Reply:
I had some hopes, but then I had to fix it. :-)